celestia/peers.txt

```diff
@@ -1 +1 @@
-
+0cb62ebbce54e2d5e72ced753a879d3c08c61583@162.19.171.80:26656
```

Comment:
please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

Reply:
Made the updates, sorry for the delay, @evan-forbes